rest-service/manager_rest/test/security/test_authorization.py

```diff
@@ -13,45 +13,492 @@
+                raise
+
+        # ...but bob, clair and dave should not
+        self.assertRaisesRegexp(UserUnauthorizedError,
+                                '401: user unauthorized',
+                                self.deployer_client.deployments.delete,
+                                'dp_example_1')
+        self.assertRaisesRegexp(UserUnauthorizedError,
+                                '401: user unauthorized',
+                                self.viewer_client.deployments.delete,
+                                'dp_example_1')
+        self.assertRaisesRegexp(UserUnauthorizedError,
+                                '401: user unauthorized',
+                                self.simple_user_client.deployments.delete,
+                                'dp_example_1')
+
+    def test_execution_operations(self):
+        self.admin_client.blueprints.upload(
+            self.blueprint_path, 'blueprint_1')
+        self.admin_client.deployments.create('blueprint_1', 'deployment_1')
+        self.admin_client.blueprints.upload(
+            self.blueprint_path, 'blueprint_2')
+        self.admin_client.deployments.create('blueprint_2', 'deployment_2')
+        # admins, deployers and viewers should be able so list executions
+        executions_list = self.admin_client.executions.list()
+        self.assertEqual(len(executions_list), 2)
+        executions_list = self.deployer_client.executions.list()
+        self.assertEqual(len(executions_list), 2)
+        executions_list = self.viewer_client.executions.list()
+        self.assertEqual(len(executions_list), 2)
+        # ...but simple users should not
+        self.assertRaisesRegexp(UserUnauthorizedError,
+                                '401: user unauthorized',
+                                self.simple_user_client.executions.list)
+
+        self.wait_for_deployment_creation(self.admin_client, 'deployment_1')
+        self.wait_for_deployment_creation(self.admin_client, 'deployment_2')
+
+        # admins and deployers should be able to start executions...
+        execution1 = self.admin_client.executions.start(
+            deployment_id='deployment_1', workflow_id='install')
+        execution2 = self.deployer_client.executions.start(
+            deployment_id='deployment_2', workflow_id='install')
+        # ...but viewers and simple users should not
+        self.assertRaisesRegexp(UserUnauthorizedError,
+                                '401: user unauthorized',
+                                self.viewer_client.executions.start,
+                                'dummy_dp', 'install')
+        self.assertRaisesRegexp(UserUnauthorizedError,
+                                '401: user unauthorized',
+                                self.simple_user_client.executions.start,
+                                'dummy_dp', 'install')
+
+        # admins, deployers and viewers should be able to get executions...
+        execution_1 = self.admin_client.executions.get(execution1.id)
```

Comment:
change default configuration to allow getting a specific resource and not others